lib/utils/logger.js

```diff
@@ -0,0 +1,66 @@
+/* eslint import/no-dynamic-require: 0 */
+const os = require('os');
+const _ = require('lodash');
+const { check } = require('./environment-check');
+const type = require('./types');
+
+const {
+  createLogger,
+  format,
+  transports,
+} = require('winston');
+
+const {
+  combine,
+  timestamp,
+  printf,
+} = format;
+
+const logDirectory = `${os.homedir()}/.autolabjs`;
+
+check('LOGGERCONFIG');
+
+const config = require(process.env.LOGGERCONFIG);
+
+const logBlacklist = ['log', 'password'];
```

Comment:
Is it better to put the blacklist in the config file?

Reply:
I suppose we can have a separate file for logger's default config, say `default-logger.json` and have all these preferences in that file. That would separate out cli and logger preferences.

Reply:
A single config file might be better in this scenario.